### PR TITLE
[auth] Fix reload page after logon on unauthorized page

### PR DESCRIPTION
**Pull request by fredpook**
See #61

### CODE REVIEW

File: modules/mod_ginger_auth/mod_ginger_auth.erl [hunk 1]
```diff
@@ -41,8 +41,12 @@ manage_schema(install, Context) ->
 %% @doc Check the action template argument for post-logon continuation actions
 %% -spec observe_logon_actions(#logon_actions{}, list(), #context{}) -> undefined | list().
 observe_logon_actions(#logon_actions{args=Args}, _Acc, _Context) ->
-    %% Can probably be removed when https://github.com/zotonic/zotonic/pull/1043 is resolved.
-    proplists:get_value(action, Args).
+    case proplists:get_value(action, Args) of
+        undefined ->
+            [];
+        _ ->
+            proplists:get_value(action, Args)
```
Comment (ddeboer) [inline modules/mod_ginger_auth/mod_ginger_auth.erl]: Better:

```erlang
Value -> Value
```

**Comment by ddeboer** [comment]
👍 on the fix

See comment on how to make this a bit more legible.

File: modules/mod_ginger_auth/mod_ginger_auth.erl (hunk 1 above)
Comment (ddeboer) [inline modules/mod_ginger_auth/mod_ginger_auth.erl]: Even better perhaps: `proplists:get_value(action, Args, [])`?

**Comment by fredpook** [comment]
Can this be merged in 0.11.0 ?

**Comment by ddeboer** [comment]
Squashed and merged. @DorienD @Dirklectisch Can this one go to 0.11 too? If so, please cherry-pick onto the 0.11 branch.

**Comment by DorienD** [comment]
Yes please

**Pull request merged**